source/includes/code-examples/FindManyAsync.cs

```diff
@@ -0,0 +1,97 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class FindManyAsync
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static async Task Main(string[] args)
+    {
+        Setup();
+
+        // Find multiple documents using builders
+        WriteLine("Finding documents with builders...:");
+        var restaurantsBuilders = await FindMultipleRestaurantsBuilderAsync();
+        WriteLine("Number of documents found: " + restaurantsBuilders.Count);
+
+        // Extra space for console readability 
+        WriteLine();
+
+        // Find multiple documents using LINQ
+        WriteLine("Finding documents with LINQ...:");
+        var restaurantsLINQ = await FindMultipleRestaurantsLINQAsync();
+        WriteLine("Number of documents found: " + restaurantsLINQ.Count);
+
+        WriteLine();
+
+        // Find all documents
+        WriteLine("Finding all documents...:");
+        var allRestaurants = await FindAllRestaurantsAsync();
+        WriteLine("Number of documents found: " + allRestaurants.Count);
+    }
+
+    private static async Task<List<Restaurant>> FindMultipleRestaurantsBuilderAsync()
+    {
+        // start-find-builders-async
+        var filter = Builders<Restaurant>.Filter
+            .Eq(r => r.Cuisine, "Pizza");
+
+        return await _restaurantsCollection.Find(filter).ToListAsync();
+        // end-find-builders-async
+    }
+
+    private static async Task<List<Restaurant>> FindMultipleRestaurantsLINQAsync()
+    {
+        // start-find-linq-async
+        return await _restaurantsCollection.AsQueryable()
```

Comment:
nit: in the driver we would use this formatting:

     return await _restaurantsCollection
            .AsQueryable()
            .Where(r => r.Cuisine == "Pizza")
            .ToListAsync(); 

where each operation is placed to an own raw, but feel free to use a different format if you think it suites better

Reply:
I like this idea, but because it would need to be made across all LINQ pages, I'd like to ticket this separately. I'll make the ticket and we can implement the change after these docs are live.